exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/advanced.ipynb

```diff
@@ -0,0 +1,342 @@
+{
```

Comment:
<p>... external package repositories <strong>are</strong> not available ...</p><p> </p><p>... many logs <strong>that can help</strong> identify the problem.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>